python/tvm/auto_scheduler/task_scheduler.py

```diff
@@ -591,21 +592,21 @@ def pre_tune(self, task_scheduler, task_id):
         for i in range(len(task_scheduler.tasks)):
             id_str = f"{i}"
             latency_str = (
-                "%.3f" % (1e3 * task_scheduler.best_costs[i])
+                f"{1e3 * task_scheduler.best_costs[i]:.3f}"
                 if task_scheduler.best_costs[i] < 1e9
                 else "-"
             )
             task_desc = task_scheduler.tasks[i].desc
+            best_cost = task_scheduler.best_costs[i]
             speed_str = (
-                "%.2f"
-                % (task_scheduler.tasks[i].compute_dag.flop_ct / task_scheduler.best_costs[i] / 1e9)
+                f"{task_scheduler.tasks[i].compute_dag.flop_ct / best_cost / 1e9:.2f}"
                 if task_scheduler.best_costs[i] < 1e9
                 else "-"
             )
-            trials_str = "%d" % (task_scheduler.task_cts[i] * task_scheduler.num_measures_per_round)
+            trials_str = f"{(task_scheduler.task_cts[i] * task_scheduler.num_measures_per_round)}"
             print(
-                "| %4s | %61s | %12s | % 14s | %6s |"
-                % (id_str, task_desc, latency_str, speed_str, trials_str)
+                f"| {id_str:4s} | {task_desc:61s} | {latency_str:12s} | "
+                f"{speed_str:14s} | {trials_str:6s} |"
```

Comment:
Last thing: you are mixing loggers and print. It is more common to use loggers in such software. When developing or testing, you may need to configure logging.